cmd/curve.go

```diff
@@ -29,7 +30,11 @@ var curveCmd = &cobra.Command{
 		for _, config := range configuration.CurrentConfig.Fans {
 			if config.HwMon != nil {
 				for _, controller := range controllers {
-					if controller.Platform == config.HwMon.Platform {
+					matched, err := regexp.MatchString(config.HwMon.Platform, controller.Platform)
+					if err != nil {
+						ui.Fatal("Unable to read curve platform: %s", config.HwMon.Platform)
```

Comment:
Can we make this error message (and the others) more specific and add context please?
Like: `"Failed to match platform regex of %s (%s) against controller platform %s", config.ID, config.HwMon.Platform, controller.Platform`